src/OpenTelemetry.AutoInstrumentation/Configuration/EnvironmentConfigurationTracerHelper.cs

```diff
@@ -52,11 +52,13 @@ internal static class EnvironmentConfigurationTracerHelper
             };
         }
 
-        // Exporters can cause dependency loads.
-        // Should be called later if dependency listeners are already setup.
-        builder.SetExporter(settings, pluginManager);
+        builder
+            .SetSampler(settings)
+            // Exporters can cause dependency loads.
+            // Should be called later if dependency listeners are already setup.
+            .SetExporter(settings, pluginManager)
+            .AddSource(settings.ActivitySources.ToArray());
```

Comment:
Any reason why you refactored to fluent style? Personally, I prefer avoiding it as it is easier during debugging 😉

Reply:
Our internal API is written in the fluent style way. What is more, official SDK also, so we should follow it.